### PR TITLE
attempt to support serverless 3

### PR DESCRIPTION
**Pull request by wheresrhys**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,10 +21,11 @@
     "@financial-times/rel-engage": "^8.0.3"
   },
   "peerDependencies": {
-    "serverless-offline": "^3.0.0"
+    "serverless-offline": "^8.8.0",
```
Comment (kiyaGu) [inline package.json:24]: moving from v3 to v8 is a big jump ... could that brake something?

**Review by kiyaGu — COMMENTED** [review]
are the `console.log` left by accident

File: package.json (hunk 1 above)
Comment (wheresrhys) [reply package.json:24]: It seems to work ok. Also, this is only used in dev so the impact of any bug will be small. But I expect we will find out whether it's a problem when we try rolling this out

**Review by kiyaGu — APPROVED** [review]

File: index.js [hunk 2]
```diff
@@ -10,7 +10,7 @@ const assumeRole = async config => {
 	const AWS = require('aws-sdk'); // eslint-disable-line global-require
 	AWS.config.region = config.provider.region;
 	const sts = new AWS.STS({ apiVersion: '2011-06-15' });
-
+	console.log(`Attempting to assume AWS role ${config.provider.role}`);
 	return sts
 		.assumeRole({
 			RoleArn: config.provider.role,
```
Comment (benholmes-ft) [inline index.js]: One of the deprecation notices for Serverless v3 is to move the role into an iam property (see screenshot). What's your opinion on how to manage this? We should probably support both possibilities.

image

**Pull request merged**